plasma_framework/contracts/src/framework/ExitGameController.sol

```diff
@@ -187,9 +187,9 @@ contract ExitGameController is ExitGameRegistry {
      * @notice Checks whether any of the output with the given outputIds is already spent
      * @param _outputIds Output IDs to check
      */
-    function isAnyOutputsSpent(bytes32[] calldata _outputIds) external view returns (bool) {
+    function isAnyOutputsFinalized(bytes32[] calldata _outputIds) external view returns (bool) {
```

Comment:
Strictly speaking this should be `isAnyOutputFinalized`.
But it's not a big deal, I'll leave it up to you...